### PR TITLE
Introudce Containerfile.operator for Productized build

### PR DESCRIPTION
**Pull request by clyang82**
## Related issue(s)

Fixes https://issues.redhat.com/browse/ACM-12399

### CODE REVIEW

File: agent/Containerfile.operator [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024 Red Hat, Inc.
```
Comment (yanmxa) [inline agent/Containerfile.operator:1]: So we will keep both `Dockerfile` and `Containerfile.operator` for each component?

Is the `Containerfile.operator` only for the conflux pipeline?

Comment (clyang82) [reply agent/Containerfile.operator:1]: Dockerfile is a community build, while Containerfile.operator is for productized build.

**Pull request merged**